reference-architectures/full-stack/deploy-arch-ibm-pvs-inf-full-stack.md

```diff
@@ -51,26 +51,26 @@ content-type: reference-architecture
 {: toc-compliance="SAPCertified"}
 {: toc-version="4.2.0"}
 
-PowerVS workspace deployment of the Power Virtual Server with VPC landing zone creates VPC services and a Power Virtual Server workspace and interconnects them.
+The PowerVS workspace deployment of the Power Virtual Server with VPC landing zone creates VPC services and a Power Virtual Server workspace and interconnects them.
 
 A proxy service for public internet access from the PowerVS workspace is configured. You can optionally configure some management components on VPC (such as an NFS server, NTP forwarder, and DNS forwarder).
 
-## Architecture diagram
+## Architecture Diagram
```

Comment:
Fix the capitalization. Original is correct for this this and ext section. (Use sentence case, not Headline Case)